pkg/operator/ceph/object/rgw.go

```diff
@@ -14,39 +14,32 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package rgw for the Ceph object store.
+// Package object for the Ceph object store.
 package object
 
 import (
 	"fmt"
 
 	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
 	"github.com/rook/rook/pkg/clusterd"
-	"github.com/rook/rook/pkg/daemon/ceph/client"
-	rgwdaemon "github.com/rook/rook/pkg/daemon/ceph/rgw"
+	daemonconfig "github.com/rook/rook/pkg/daemon/ceph/config"
+	cephconfig "github.com/rook/rook/pkg/operator/ceph/config" // TODO: rename to just config
 	"github.com/rook/rook/pkg/operator/ceph/pool"
 	"github.com/rook/rook/pkg/operator/k8sutil"
-	"k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
 )
 
-const (
-	appName        = "rook-ceph-rgw"
-	keyringName    = "keyring"
-	certVolumeName = "rook-rgw-cert"
-	certMountPath  = "/etc/rook/private"
-	certKeyName    = "cert"
-	certFilename   = "rgw-cert.pem"
-)
-
-type config struct {
+type config struct { // TODO rename to 'cluster'
```

Comment:
i think i named it `config` because it seemed more applicable to object than `cluster`, and i was thinking of renaming the other `cluster` structs as `config`. But either way is fine to me.

Reply:
How about `clusterConfig`? Then it's clear that it's for the cluster (not just an individual daemon) and a config struct.